iphone/cli/commands/_build.js

```diff
@@ -4002,6 +4003,7 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 						// fall through to default case
 
 					default:
+						imageAssets[relPath] = info;
```

Comment:
I'm not sure this is a good idea. Images are going to be copied into the app twice: once as they do now and again into the asset catalog.

You should add a 'jpg' and 'png' case above it and track `imageAssets` separately. This case should be BEFORE 'html' as 'html' is designed to fall through.